sfm_pc/management/commands/import_google_doc.py

```diff
@@ -117,27 +122,33 @@ def get_credentials(self):
 
     def disconnectSignals(self):
         from django.db.models.signals import post_save
-        from sfm_pc.signals import update_organization_index, \
-            update_person_index, update_violation_index, \
-            update_membership_index, update_composition_index
+        from complex_fields.base_models import object_ref_saved
+        from sfm_pc.signals import (
+            update_organization_index, update_person_index, update_violation_index,
+            update_membership_index, update_composition_index, update_source_index
+        )
 
-        post_save.disconnect(receiver=update_organization_index, sender=Organization)
-        post_save.disconnect(receiver=update_person_index, sender=Person)
-        post_save.disconnect(receiver=update_violation_index, sender=Violation)
-        post_save.disconnect(receiver=update_membership_index, sender=MembershipPerson)
-        post_save.disconnect(receiver=update_composition_index, sender=Composition)
+        object_ref_saved.disconnect(receiver=update_organization_index, sender=Organization)
```

Comment:
For each signal (dis)connection, we were (dis)connecting the wrong signal (`post_save` is the builtin Django save signal, whereas `object_ref_saved` is the `complex_fields` version). The tests surfaced this discrepancy.